lib/generators/database_consistency/database_consistency_generator.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# Basic generator to copy useful defaults for Rails apps
+class DatabaseConsistencyGenerator < Rails::Generators::Base
```

Comment:
`database_consistency` doesn't depend  on `Rails`, only on `ActiveRecord`. What do you think to add some installer instead? Such as `bundle exec database_consistency install` or similar?

Reply:
If you still want to go with this one, please make sure you check if `Rails::Generators::Base` is available.

Reply:
I've replaced the Rails generator with an install command in bin/database_consistency, removing the dependency on Rails generators.

If you find this clutters the bin file we could move this code to another file. Or revert to using the Rails generator but check that the `Rails::Generators::Base` class is available, as you recommend.